Persistence/Entities/Sender.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace shipping_service.Persistence.Entities
+{
+    public class Sender
+    {
+        public ulong Id { get; set; }
+        public byte[] HashedPassword { get; set; }
+        public DateTime Created { get; set; }
```

Comment:
Not sure why we need Created and Modified fields (does modified means changed password?) but it does not hurt to have them.

Reply:
It's standard practice to have Created (when this entity was created in DB) and Modified (when this entity was last modified in DB).